bin/tachyon-mount.sh

```diff
@@ -213,6 +213,7 @@ case "${1}" in
         mount_local $1
         ;;
       workers)
+        bin=`cd "$( dirname "$0" )"; pwd`
         $LAUNCHER $bin/tachyon-workers.sh $bin/tachyon-mount.sh $1
```

Comment:
Nice @jsimsa , there are many references to `$bin` in other `bin/tachyon*` files, do you want me to modify all of them as you suggested ( `$bin` to `${BIN}` ), to make a standard under those shells ?